run.sh

```diff
@@ -86,7 +89,7 @@ VIN_LIST=$(echo $VIN_LIST | sed -e 's/[|,;]/ /g')
 vin_count=0
 for vin in $VIN_LIST; do
   # Populate BLE Local Names list
-  vin_count=$(expr $vin_count + 1)
+  vin_count=$((vin_count + 1))
```

Comment:
Somehow this works, not sure why it wasn't working for the counter in main loop?!  Probably written differently?